app/views/MultipleQuestionView.scala.html

```diff
@@ -93,7 +93,7 @@ <h1 class="govuk-heading-l govuk-!-margin-bottom-3">@renderMessage(viewModel.hea
         .withId(s"value.${index + 1}")
         .asNumeric()
         .withHint(field.hint.map(h => Hint(content = HtmlContent(renderMessage(h)))))
-        .withWidth(width.fold[InputWidth](InputWidth.Fixed10)(identity))
+        .withWidth(width.getOrElse(InputWidth.Fixed4))
```

Comment:
Just to double check, would this default `Fixed4` work for all these where the value is not explicitly passed in:
![image](https://github.com/user-attachments/assets/00911b69-1842-4b93-9328-dcc0eb43c22e)

Reply:
I have checked, It works for the numeric only fields so other fields are not affected